server/main.ts

```diff
@@ -1,9 +1,9 @@
-const app = require('./server.js');
-const logger = require('./logger.js');
-const { dbClient } = require('./db');
-const keycloak = require('./keycloak');
+import { app } from './server';
+import logger from './logger';
+import { dbClient } from './db';
+import keycloak from './keycloak';
 
-const port = process.env.SERVER_PORT || 8080;
+const port = Number(process.env.SERVER_PORT) || 8080;
 
```

Comment:
Curious why `process.env.SERVER_PORT` is now a string? or why Number() is necessary

Reply:
Anything in `process.env` can be assumed to be a string. While it technically works, you shouldn't use a string as a port, and TS will yell at you for it. Casting to a number ensures it's actually a number.